onefit/frontend/src/components/workouts/Form.js

```diff
@@ -1,13 +1,13 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
 
 export class Form extends Component {
-    render() {
-        return (
-            <div>
-                <h1>Add Workout Form</h1>
-            </div>
-        )
-    }
+  render() {
```

Comment:
might want to look into a linter

Reply:
I have a python linter, can't seem to find a linter for react